core/src/main/java/org/eclipse/hono/config/ClientConfigProperties.java

```diff
@@ -57,6 +61,7 @@
     private int connectTimeoutMillis = DEFAULT_CONNECT_TIMEOUT;
     private String credentialsPath;
     private long flowLatency = DEFAULT_FLOW_LATENCY;
+    private int heartbeatIntervalMillis = DEFAULT_HEARTBEAT_INTERVAL;
```

Comment:
why don't we call it *idleTimeout* as the AMQP spec and QPid do?

Reply:
Ok, yes, makes sense to focus on the idleTimeout rather than the heartbeat. 

I would use the _actual_ idleTimeout here after which the connection gets closed.
The `idle-time-out` in the `open` frame and the heartbeat would be half of that then.

Reply:
I've amended the commit accordingly.